uPlayerConfig.yml

```diff
@@ -0,0 +1,55 @@
+#** *****************************************************************************
```

Comment:
create subdirectory for `configs` or one specifically for `playerConfigs`

Reply:
done